api/v1/paths/upload.js

```diff
@@ -0,0 +1,48 @@
+
+module.exports = function () {
+    let operations = {
+        POST,
+    };
+
+    async function POST(req, res, next) {
+        try {
+            const files = Object.assign([] ,req.files)
```

Comment:
This part is a bit strange, lets make the handling of the different types of `files` more explicit.